### PR TITLE
1465 launchable webapp from its own landing page

### PR DESCRIPTION
**Pull request by zhiyuli**
This pr is to address #1465

### CODE REVIEW

File: hs_tools_resource/tests/test_web_app.py [hunk 1]
```diff
@@ -152,6 +154,37 @@ def test_web_app_res_specific_metadata(self):
                                          element_id=ToolIcon.objects.first().id)
         self.assertEqual(ToolIcon.objects.all().count(), 0)
 
+        ####################
```
Comment (pkdash) [inline hs_tools_resource/tests/test_web_app.py]: I would prefer to remove this.

File: hs_tools_resource/tests/test_web_app.py [hunk 2]
```diff
@@ -152,6 +154,37 @@ def test_web_app_res_specific_metadata(self):
...
+        # Class: AppHomePageUrl
+        ####################
```
Comment (pkdash) [inline hs_tools_resource/tests/test_web_app.py]: As before.

File: hs_tools_resource/tests/test_web_app.py [hunk 3]
```diff
@@ -152,6 +154,37 @@ def test_web_app_res_specific_metadata(self):
...
+        # verify no AppHomePageUrl obj
+        self.assertEqual(AppHomePageUrl.objects.all().count(), 0)
+
+        # create 1 AppHomePageUrl obj with required params
+        resource.create_metadata_element(self.resWebApp.short_id,
+                                         'AppHomePageUrl',
+                                         value='https://my_web_app.com')
+        self.assertEqual(AppHomePageUrl.objects.all().count(), 1)
+
+        # may not create additional instance of ToolIcon
```
Comment (pkdash) [inline hs_tools_resource/tests/test_web_app.py]: Comment seems need change.

**Comment by pkdash** [comment]
@zhiyuli Can you make the hs_tools_resource app go through pylint checks by removing all statements related to this app from the  setup.cfg file? Thanks.

**Comment by zhiyuli** [comment]
@pkdash 
your comments have been addressed; pylint has been enabled. Thanks

**Comment by pkdash** [comment]
+1

**Pull request merged**